code/api/src/modules/user/model.js

```diff
@@ -18,6 +18,18 @@ module.exports = function(sequelize, DataTypes) {
     description: {
       type: DataTypes.STRING
     },
+    address: {
```

Comment:
Like how you split up the address into separate columns, it makes it more accurate